providers/snowflake/docs/changelog.rst

```diff
@@ -27,6 +27,17 @@
 Changelog
 ---------
 
+7.0.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+.. warning::
+   The following breaking changes were introduced:
+   * ``private_key_content``  in Snowflake connection should now be base64 encoded.
```

Comment:
We should explain how to migrate from 6.x -> 7

Reply:
I gave the helper code in the docs. Should i just link it here or write here as well?

Reply:
Change log should provide exactly the info needed to migrate.
Docs are not about migration they are about the current state (user who starts with updated version)

Reply:
@eladkal , I updated it. Please let me know if the update looks good

Reply:
I am not confident this is a breaking change.
This feels more like a bug fix.
Correct me if I am wrong but starting the commit that "broke" it, this never worked as intended? Is there any production environment out there that the current code works for them thus changing it to fix the bug will cause a breaking change?

Reply:
You are correct that an earlier commit which marked the connection field as private broke it. I don't think it worked after that. But, customer's still need to update the keys to be base64 encoded. If you don't think this is a breaking change, I'll bump the minor version

Reply:
The changelog entry is actually good. No need to remove it.
Just not with the breaking version title :)

Reply:
@eladkal , So, I bumped the minor version and added a note. I did not mention the PR specifically. Should I add this PR to the changelog, or will that be taken care of during the provider release?

Reply:
No need
I pick it up during release